src/common/utxobased/db/newProcessor.ts

```
@@ -161,26 +194,160 @@ export async function makeNewProcessor(
+              '',
+              [txId]
+            )
+            txs.push(...txById)
+          } catch {}
+        }
+        // Fetch transactions by min block height
+        if (blockHeightMax != null && blockHeight == null) blockHeight = 0
+        if (blockHeight != null) {
+          try {
+            const txidsByMinBlockHeight = await (
+              await tables.txIdsByBlockHeight.query(
+                '',
+                blockHeight,
+                blockHeightMax
+              )
+            )
+              .reverse()
+              .map(({ [RANGE_ID_KEY]: id }) => id)
+
+            const txsById: IProcessorTransaction[] = await tables.txById.query(
+              '',
+              txidsByMinBlockHeight
+            )
+            txs.push(...txsById)
+          } catch {}
+        }
+        if (options != null) {
+          try {
+            const {
+              startEntries,
+              startIndex,
+              startDate = new Date(0),
+              endDate = new Date()
+            } = options
+
+            // Fetch transaction IDs ordered by date
+            let txData: TxsByDate
+            if (startEntries != null && startIndex != null) {
+              txData = await tables.txsByDate.queryByCount(
+                '',
+                startEntries,
+                startIndex
+              )
+            } else {
+              txData = await tables.txsByDate.query(
+                '',
+                startDate.getTime(),
+                endDate.getTime()
+              )
+            }
+            const txIdsByOptions = await txData
+              .reverse()
+              .map(({ [RANGE_ID_KEY]: id }) => id)
+
+            const txsByOptions = await tables.txById.query('', txIdsByOptions)
+            // Make sure only existing transactions are included
+            txs.push(...txsByOptions.filter(tx => tx != null))
+          } catch {}
```

Fix silent fail here too